### PR TITLE
Windows kalite command

### PR DESCRIPTION
**Pull request by benjaoming**

### CODE REVIEW

File: kalite/version.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # THIS IS USED BY settings.py.  NEVER import settings.py here; hard-codes only!
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "dev2"
+PATCH_VERSION = "dev3"
```
Comment (aronasorman) [inline kalite/version.py]: I'm curious on why these versions are being incremented.

**Comment by aronasorman** [comment]
I don't know why you're not putting a description man :(

**Comment by benjaoming** [comment]
I don't know what happened, I wasn't posting this, I think there's something with my ENTER key :) I just have lots of tabs and windows.

File: setup.py [hunk 2]
```diff
@@ -20,19 +22,26 @@
...
+    req_pattern = re.compile(r'^\s*([^\#]+)')
+
     m = req_pattern.search(req)
     if m:
         return m.group(0).replace(" ", "")
 
+
+# Filter out comments from requirements
 DIST_REQUIREMENTS = map(filter_requirement_statements, DIST_REQUIREMENTS)
 DIST_REQUIREMENTS = filter(lambda x: bool(x), DIST_REQUIREMENTS)
```
Comment (aronasorman) [inline setup.py]: Much functional!

File: kalite/version.py (hunk 1 above)
Comment (benjaoming) [inline kalite/version.py]: Because you can't overwrite existing uploads to PyPi, and I want to add more tests.

**Comment by benjaoming** [comment]
Merging because it only touches setup.py, a file that fellow coders will become more familiar with over time... but not just before code freeze :)

**Comment by aronasorman** [comment]
I concur.

**Pull request merged**